src/pages/ProductDetail/ProductDetail.scss

```
@@ -0,0 +1,60 @@
+@use '../../styles/baseStyle/colors.scss' as color;
+@use '../../styles/baseStyle/colors.scss' as font;
+
+#slideShow {
+  width: 560px;
+  height: 700px;
+  position: relative;
+  margin: 50px auto;
+  overflow: hidden;
+  /* 리스트로 사진 5개가 쭉 나열되어있는 상태에서 사진 한개만 보일 수 있게 설정해줌.
+  560px 밖으로 튀어 나간 이미지들은 hidden으로 숨겨줘야됨*/
+
+  .slides {
+    display: flex;
+    // position: absolute;
+    // transform: translateX(-1120px);
+    // left: 0;
+    // top: 0;
+    // width: 2800px;
+    /* 슬라이드할 사진과 마진 총 넓이 */
+    transition: all 0.5s ease;
+    /*ease-out: 처음에는 느렸다가 점점 빨라짐*/
+
+    .slideContent img:hover {
+      cursor: pointer;
+      transform: scale(1.3);
+      transition: transform 0.65s;
+    }
+  }
+
+  .slideButton {
+    display: flex;
+    justify-content: space-between;
+    position: relative;
+    background: none;
+    padding: 0 20px;
+    bottom: 50%;
+  }
+
+  .slideButton button {
+    background: none;
+    font-size: 30px;
+    font-weight: 400;
+    color: color.$lightGrey;
+    opacity: 0.5;
+    cursor: pointer;
+  }
+
+  .prev:hover {
+    transform: translateX(-6px);
+    transition: all 1s ease;
+    opacity: 1;
+  }
+
+  .next:hover {
+    transform: translateX(6px);
+    transition: all 1s ease;
+    opacity: 1;
+  }
```

```suggestion
  .prev:hover, .next:hover {
    transform: translateX(-6px);
    transition: all 1s ease;
    opacity: 1;    
  }
```
이런 식으로 쓸 수 있을거 같은데 한번 해보시겠어요?

---

오른쪽 버튼은 +6, 왼쪽 버튼은 -6 만큼 작동하려고 만든건데, transform 부분 외 나머지는 묶어서 작성하면 될까요?!